Remotion/Web/ClientScript/Scripts/Utilities.ts

```diff
@@ -363,4 +363,75 @@ class WebServiceUtility
         }
       });
   };
+}
+
+class ElementResolverUtility
+{
+  // Resolves any provided css selector, checks the element for null and returns it.
+  public static ResolveSingle<TElement extends Element> (selectorOrElement: CssSelectorOrElement<TElement>, context?: ParentNode): TElement
+  {
+    if (TypeUtility.IsString(selectorOrElement))
+    {
+      const queryContext = context ? context : window.document;
+      const resolvedElement = queryContext.querySelector(selectorOrElement) as TElement;
+      if (!resolvedElement)
+        throw ('Error: Cannot find an element specified by selector "' + selectorOrElement + '".');
+
+      return resolvedElement;
+    }
+    else if (TypeUtility.IsObject(selectorOrElement))
+    {
+      ArgumentUtility.CheckNotNull("selectorOrElement", selectorOrElement);
+      return selectorOrElement;
+    }
+    else
+    {
+      throw ('Error: The type of parameter "selectorOrElements" is "' + (typeof selectorOrElement) + '" but "string" or "object" was expected.');
+    }
+  }
+
+  // Resolves any provided css selector, and returns the found elements as an array.
+  public static ResolveMultiple<TElement extends Element> (selectorOrElements: CssSelectorOrElements<TElement>, context?: ParentNode): TElement[]
+  {
+    if (TypeUtility.IsString(selectorOrElements))
```

Comment:
Please move the argumentcheck to the beginning of the method. NULL's never an option.